dockerfiles/ci/buster/Dockerfile

```diff
@@ -81,22 +82,24 @@ RUN set -eux; \
     \
 # Allow nginx to be run as non-root for tests
     chown -R circleci:circleci /var/log/nginx/ /var/lib/nginx/; \
-# JSON parser
-    export JQ_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/jq-latest"; \
-    curl --silent --show-error --location --fail --retry 3 --output /usr/bin/jq $JQ_URL; \
-    chmod +x /usr/bin/jq; \
-    jq --version; \
-    \
+# Install CMake
```

Comment:
Heads up, there's a new CMake release. You'll want something more like:
```
CMAKE_VERSION="3.20.1"
CMAKE_SHA256="b8c141bd7a6d335600ab0a8a35e75af79f95b837f736456b5532f4d717f20a09"
curl -OL https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz
(echo "${CMAKE_SHA256} cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz" | sha256sum -c -)
mkdir -p /opt/cmake/${CMAKE_VERSION}
tar --strip-components 1 -C /opt/cmake/${CMAKE_VERSION} -xf /tmp/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz;
```

Reply:
Thanks. This is updated.